Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverInBetween.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// InBetween solver positions an object in-between two tracked transforms.
+    /// </summary>
+    public class SolverInBetween : Solver
+    {
+        [SerializeField]
+        [Tooltip("Distance along the center line the object will be located. 0.5 is halfway, 1.0 is at the second transform, 0.0 is at the first transform.")]
```

Comment:
should we add [Range(0f, 1f)]?

Reply:
Did we add the range?

Reply:
Yep!
https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2245/commits/3774baa37615255b4c92bc7b57345e9e15aac44e#diff-e39ae4e27a8b2d933242b6b4fbaf451cR15

Reply:
Weird it's not showing up in the diff.

Reply:
I see it in the full diff, but it's probably not showing up here because the comment is on the line above it.

![image](https://user-images.githubusercontent.com/3580640/41750074-ca4c44c2-756e-11e8-9d0c-ca1a899e43ff.png)